Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This file describes the standard way to build libcompose, using docker
-FROM golang:1.4.2-cross
+FROM golang:1.5.1
```

Comment:
I wondered a bit about that, but the more explicit we are, the more *sure* we are that we build the same thing ; using `1.5` only would mean some could build it using 1.5.1 and the other 1.5.2, etc.. but as you said, I don't think we expect it to break with minor release so I'm fine with both :stuck_out_tongue:.